Src/AutoFakeItEasy.FakeItEasy3UnitTest/AutoFakeItEasy.FakeItEasy3UnitTest.csproj

```diff
@@ -12,6 +12,8 @@
     <!--  Suppress warning about invalid dependency version in Castle.Core.
           That is FakeItEasy dependency and we cannot fix that somehow. -->
     <NoWarn>$(NoWarn);NU1603</NoWarn>
+
+    <DefineConstants>CAN_FAKE_DELEGATES</DefineConstants>
```

Comment:
BTW, just noticed - it's better to write like this:
```xml
<DefineConstants>$(DefineConstants);CAN_FAKE_DELEGATES</DefineConstants>
```

We already define some global constants in the [Common.props](https://github.com/AutoFixture/AutoFixture/blob/9121cedad8b8eddc8993552ba87a06fbc92f97be/Src/Common.props#L25), so this line will rewrite them, rather than append.

Probably, it doesn't make difference for this project as we don't use other constants, but it would be safer to replace that..

Let me know if you wish me to force push and do that 😉

Reply:
:blush: You shouldn't have to. I'll make the change.

Reply:
Wow. That "blush" emoji looks odd.